app/route_rangers_api/tests/test_ingest_census.py

```diff
@@ -0,0 +1,158 @@
+import os
```

Comment:
@benjaleivas if you could take a look at this file, seems like a lot of the command line functionality is no longer relevant so lmk if I should create follow up tickets for subsequent test coverage or if this is sufficient for now.